components/tab-panel/index.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import { partial, noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+// import './style.scss';
+import { Component } from '@wordpress/element';
+import { default as withInstanceId } from '../higher-order/with-instance-id';
+import { NavigableMenu } from '../navigable-container';
+
+class TabButton extends Component {
```

Comment:
Could this be a functional component instead?

Reply:
I don't this is a good idea. This locks the `tabId` prop. Which can't be changed any more. Ongoing functional components debate :)

Reply:
Fair point. 😄